src/handlers/testing/exec/index.ts

```diff
@@ -102,39 +85,104 @@ class RunManager {
   }
 
   private async post<T>(path: string, body?: unknown): Promise<T> {
-    const resp = await fetch(`https://api.autoblocks.ai${path}`, {
+    const subpath = this.isCI ? '/testing/ci' : '/testing/local';
+    const resp = await fetch(`https://api.autoblocks.ai${subpath}${path}`, {
```

Comment:
Should we be storing this domain in a constant?

Reply:
Sure I'll update